src/Libraries/CoreNodes/List.cs

```diff
@@ -683,6 +684,72 @@ public static IList Transpose(IList lists)
             return transposedList;
         }
 
+        private static bool IsNull(IList list)
```

Comment:
Can we expose this method as a `public` node as well as I don't think there is a direct way to find out if an input list is `null` or not?